docs/samples/python/Integrating with external services.ipynb

```diff
@@ -0,0 +1,149 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "mature-break",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from presidio_analyzer import AnalyzerEngine\n",
+    "from text_analytics.example_text_analytics_recognizer import TextAnalyticsEntityCategory, TextAnalyticsRecognizer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "thick-separate",
+   "metadata": {},
+   "source": [
+    "1. Define which entities to get from Text Analytics"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "super-jaguar",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ta_entities = [\n",
+    "    TextAnalyticsEntityCategory(name=\"Person\",\n",
+    "                                entity_type=\"NAME\",\n",
+    "                                supported_languages=[\"en\"]),\n",
+    "    TextAnalyticsEntityCategory(name=\"Age\",\n",
+    "                                entity_type=\"AGE\",\n",
+    "                                subcategory = \"Age\", \n",
+    "                                supported_languages=[\"en\"]),\n",
+    "    TextAnalyticsEntityCategory(name=\"InternationlBankingAccountNumber\",\n",
+    "                                entity_type=\"IBAN\",\n",
+    "                                supported_languages=[\"en\"])]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "thermal-liverpool",
+   "metadata": {},
+   "source": [
+    "For a full list of entities: https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/named-entity-types?tabs=personal"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "competent-probe",
+   "metadata": {},
+   "source": [
+    "2. Instantiate the remote recognizer object (In this case `TextAnalyticsRecognizer`)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "simple-fundamentals",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "text_analytics_recognizer = TextAnalyticsRecognizer(\n",
+    "        text_analytics_key=\"<YOUR_TEXT_ANALYTICS_KEY>\",\n",
+    "        text_analytics_endpoint=\"<YOUR_TEXT_ANALYTICS_ENDPOINT>\",\n",
+    "        text_analytics_categories = ta_entities)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "novel-mission",
+   "metadata": {},
+   "source": [
+    "3. Add the new recognizer to the list of recognizers and run the `PresidioAnalyzer`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "permanent-samuel",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "analyzer = AnalyzerEngine()\n",
+    "analyzer.registry.add_recognizer(text_analytics_recognizer)\n",
+    "\n",
+    "results = analyzer.analyze(\n",
+    "    text=\"David is 30 years old. His IBAN: IL150120690000003111111\", language=\"en\"\n",
```

Comment:
The return of David 😆

Reply:
This works even with David involved :) so leaving it here.